odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1687,34 +1607,146 @@
-            <field name="currency_subunit_label">Centavo</field>
+            <field name="currency_unit_label">Cuban convertible peso</field>
+        </record>
+
+        <record id="VES" model="res.currency">
+            <field name="name">VES</field>
+            <field name="full_name">Venezuelan bolívar soberano</field>
+            <field name="symbol">Bs</field>
+            <field name="rounding">0.01</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="UYW" model="res.currency">
+            <field name="name">UYW</field>
+            <field name="full_name">Unidad previsional</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.0001</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">peso</field>
+            <field name="currency_subunit_label">centésimo</field>
+        </record>
+
+        <record id="UYI" model="res.currency">
+            <field name="name">UYI</field>
+            <field name="full_name">Uruguay Peso en Unidades Indexadas</field>
+            <field name="symbol">$</field>
+            <field name="rounding">1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Peso</field>
+            <field name="currency_subunit_label">centésimo</field>
+        </record>
+
+        <record id="USN" model="res.currency">
+            <field name="name">USN</field>
+            <field name="full_name">United States dollar (next day)</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Dollars</field>
+            <field name="currency_subunit_label">Cents</field>
+        </record>
+
+        <record id="STN" model="res.currency">
+            <field name="name">STN</field>
+            <field name="full_name">São Tomé and Príncipe dobra</field>
+            <field name="symbol">Db</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Dobra</field>
+            <field name="currency_subunit_label">cêntimo</field>
+        </record>
+
+        <record id="MXV" model="res.currency">
+            <field name="name">MXV</field>
+            <field name="full_name">Mexican Unidad de Inversion</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Pesos</field>
+            <field name="currency_subunit_label">Centavos</field>
+        </record>
+
+        <record id="CHW" model="res.currency">
+            <field name="name">CHW</field>
+            <field name="full_name">WIR franc (complementary currency)</field>
```

Comment:
```suggestion
            <field name="full_name">WIR franc</field>
```

Reply:
Not sure about this one.. Let's see what @mart-e has to say about it.

Reply:
As much as I love the ideology of local monetary systems, I don't think this is useful to add them in the base list and is the open door to all the city-based currencies or worse... cryptocurrencies :scream: 
I could see that in l10n_ch if somebody is asking.